Scripts/pycairoInstall.py

```diff
@@ -0,0 +1,45 @@
+#This script install pycairo.
+import platform
+import os
+import sys
+import urllib.request
+
+#python 3.7 32-bit
+#pycairo‑1.19.1‑cp37‑cp37m‑win32.whl
+if 'Windows' in platform.system():
+    if sys.version[:3]=='3.7' and platform.machine()=='x86':
+        urllib.request.urlretrieve("https://download.lfd.uci.edu/pythonlibs/s2jqpv5t/pycairo-1.19.1-cp37-cp37m-win32.whl", "pycairo-1.19.1-cp37-cp37m-win32.whl")
+        print("Sucessfully downloaded Cairo for your system")
+        print("Installing Cairo")
+        os.system("pip install pycairo-1.19.1-cp37-cp37m-win32.whl")
+        os.remove("pycairo-1.19.1-cp37-cp37m-win32.whl")
+        print("Succesfully installed Cairo")
+
+    #python 3.7 AMD64(64-bit)
+    #pycairo-1.19.1-cp37-cp37m-win_amd64.whl
+    elif sys.version[:3]=='3.7' and platform.machine()=='AMD64':
+        urllib.request.urlretrieve("https://download.lfd.uci.edu/pythonlibs/s2jqpv5t/pycairo-1.19.1-cp37-cp37m-win_amd64.whl", "pycairo-1.19.1-cp37-cp37m-win_amd64.whl")
```

Comment:
How safe is it to download wheel files from this url?

Reply:
This is the URL of the actual file downloaded to install by a windows *user*.
Linked from ReadMe.
https://www.lfd.uci.edu/~gohlke/pythonlibs/#pycairo

Reply:
Or, can we add the `.whl` to the repo and pull from the repo while installing.

Reply:
> Or, can we add the `.whl` to the repo and pull from the repo while installing.

I don't think it's good to host files of other libraries on our repo just for the sake of installation. It would open up many other questions from users like _"Why isn't x dependency hosted as well?"_

Reply:
Also, that means we have to keep updating the file in our repo to the latest one, which of course we don't want to add to our already large to-do list.

Reply:
So, pulling from that website is good.